tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -1711,6 +1711,37 @@ def test_OnDiskDataset_load_tasks():
         dataset = None
 
 
+def test_OnDiskDataset_load_all_nodes_set():
+    """Test preprocess of OnDiskDataset."""
+    with tempfile.TemporaryDirectory() as test_dir:
+        # All metadata fields are specified.
+        dataset_name = "graphbolt_test"
+        num_nodes = 4000
+        num_edges = 20000
+        num_classes = 10
+
+        # Generate random graph.
+        yaml_content = gbt.random_homo_graphbolt_graph(
```

Comment:
add dedicated test case for heterograph as well.

Reply:
I begin to wonder whether it is necessary to write a new function `test_OnDiskDataset_load_all_nodes_set` to test the new attribute `all_nodes_set`. Because several testors above with prefix `test_OnDiskDataset_load` are designed to test modify same contend of the dataset. However there is nothing to modify for `all_nodes_set`. Mayby I should remove this test function and do the testing in `test_OnDiskDataset_Graph_homogeneous` and `test_OnDiskDataset_Graph_heterogeneous`?